src/server/app.ts

```diff
@@ -99,6 +100,18 @@ export const createApp = async () => {
     app.use(express.static(PUBLIC_DIR, { maxAge: "180 days" }));
   }
 
+  // Serve service worker from root for complete SW scope
+  app.get("/service-worker.js", (req, res) =>
+    res.sendFile("service-worker.js", { root: config.ASSETS_DIR })
+  );
+
+  // Serve offline page
+  app.get("/offline.html", (req, res) =>
+    res.sendFile("offline.html", {
```

Comment:
How is the file being copied over? I think webpack should copy over this file to the clients build directly. This will allow the clients build directory to later be uploaded to CDN more easily allowing static files to be available via CDN.

Reply:
Yeah, it's a little out of scope of this PR, but I'd rather do things the right way with little scope creep, than make half way changes we'll want to fix later on, especially since we know we want to make those changes in the not so distant future.